### ISSUE TITLE
menuByName() does not return translated text

### ISSUE THREAD

**Issue by JohnAlbin** [drupal-graphql/graphql#773]
With a fresh install of the umami install profile (8.7) with English and Spanish content, I would expect `menuByName(name: "main", language: ES)` to return Spanish text.

However, the returned data contains translated URLs but none of the text fields are translated.

e.g.

Requesting this:
```
{
  menuByName(name: "main", language: ES) {
    name
    description
    links {
      label
      url {
        path
      }
    }
  }
}
```

returns this data:

```
{
  "data": {
    "menuByName": {
      "name": "Main navigation",
      "description": "Site section links",
      "links": [
        {
          "label": "Home",
          "url": {
            "path": "/es"
          }
        },
        {
          "label": "Articles",
          "url": {
            "path": "/es/articles"
          }
        },
        {
          "label": "Recipes",
          "url": {
            "path": "/es/recipes"
          }
        }
      ]
    }
  }
}
```

So the following fields are being returned in the default language and not in the language requested:
* menu name
* menu description
* menu link label ( of menu links provided by modules )

FYI, custom menu link labels are translated correctly by graphQL.

**Comment by JohnAlbin** [comment]
FYI, here's how Umami has the menus translated:

<img width="804" alt="Screen Shot 2019-06-10 at 15 46 47" src="https://user-images.githubusercontent.com/33429/59180304-28a90980-8b97-11e9-8a1c-81fa4c47d9c1.png">
<img width="792" alt="Screen Shot 2019-06-10 at 15 47 36" src="https://user-images.githubusercontent.com/33429/59180305-2941a000-8b97-11e9-9758-f6dbf5f0d74e.png">

**Comment by JohnAlbin** [comment]
I just checked and umami doesn't have custom menu links translation turned on. That makes sense since the bundled menu links are controlled by umumi's modules and are not custom menu links like a normal website where content editors are creating menu links.

I tried adding a custom menu link to umami and GraphQL does translate that text. I'll update the ticket description to better indicate what is not working.
